src/components/public-service-details-page/index.tsx

```diff
@@ -516,6 +535,45 @@ const PublicServiceDetailsPage: FC<Props> = ({
             </ContentSection>
           )}
 
+          {isGroupedBy.length > 0 && eventsMap && (
```

Comment:
eventsMap er ikke beste sjekk i dette tilfellet fordi en tom map gir false positive. Bruk kanskje Object.values(eventsMap) eller noe lignende for å sjekke at den ikke er tom

Reply:
Du har uansett en fallback til uri hvis den ikke finnes i eventsMap, så kanskje du ikke trenger test på eventsMap i det hele tatt.